pom.xml

```diff
@@ -115,6 +115,23 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
I think this is not connected to the PR main goal.
I'm not sure this is needed and that it requires nothing extra, so it is better to separate it to another PR with specific description.

Reply:
Moved to separate PR